tags.json

```diff
@@ -3257,6 +3257,17 @@
   "phone-outgoing": [
     "call"
   ],
+  "picture-in-picture-active": [
```

Comment:
I think these should be just two alternatives not "states", so no `picture-in-picture-active`, just `picture-in-picture` and  `picture-in-picture-2`.